Program.cs

```diff
@@ -56,22 +77,10 @@ static void Main(string[] args)
                     }
                     Console.WriteLine("Oh, forgot to mention YOU'RE FIRED!!!");
                 }
-                else if (response.ToLower() == "no")
-                {
-                    Console.WriteLine("Are you sure about that?");
-                    response = Console.ReadLine();
-
-                    if (response.ToLower() == "yes")
-                    {
-                        // making sure 
-                    }
-                    Console.WriteLine("YOU'RE FIRED!!");
 
-                   
-                    
-                }
 
             }
```

Comment:
The scope that this curly brace closes on line 82 (which was opened on line 9) isn't necessary.  It's not part of a method or if-statement and could be removed